widgets/range-slider/range-slider.js

```diff
@@ -95,6 +94,7 @@ function rangeSlider({
         templates
       });
 
+      var Slider = autoHide(headerFooter(require('../../components/Slider/Slider')));
```

Comment:
Why move it from the top of the file? It will reinstantiate and recompose each time there is a render, no?